flipperdevices/flipperzero-firmware
# Fix USB-UART bridge exit screen stopping the bridge prematurely

# What's new
  - Fixed a bug where the "Exit USB-UART?" confirmation screen would stop the bridge prematurely, without waiting for the user's input. Fix #3882
  - Refactored related code

# Verification 
  - Open the USB-UART bridge
  - Configure bridge to use channel 1
  - Press the back key
  - Verify that the "Exit USB-UART" screen shows up but that the virtual COM port remains attached
  - Verify that selecting "Stay" returns to the main screen
  - Press the back key again
  - Verify that selecting "Exit" returns to the main GPIO screen and disposes of the virtual COM port

# Checklist (For Reviewer)
